src/main/java/fivium/pat/graphql/queryfields/superuser/AddCliniciansQF.java

```diff
@@ -26,7 +26,7 @@
 @Deprecated
 public class AddCliniciansQF extends PAT_BaseQF {
 
-	private static final String ADD_CLINICIAN_PREPARED_SQL_QUERY = "INSERT INTO rns_internal.clinicians (Email, Password, Firstname, Lastname, Token, Role) VALUES (?,?,?,?,?,?)";
+	private static final String ADD_CLINICIAN_PREPARED_SQL_QUERY = "INSERT INTO clinicians (Email, Password, Firstname, Lastname, Token, Role) VALUES (?,?,?,?,?,?)";
```

Comment:
Should also tidy up fetchData(..) method so it only has 1 try/catch and doesn't return the server specific error to the client.  See CliniciansChangePasswordQF for an example